pandas/core/arrays/masked.py

```diff
@@ -1256,3 +1259,19 @@ def all(self, *, skipna: bool = True, **kwargs):
                 return result
             else:
                 return self.dtype.na_value
+
+    def _accumulate(
+        self, name: str, *, skipna: bool = True, **kwargs
+    ) -> BaseMaskedArray:
+        data = self._data
+        mask = self._mask
+
+        if name in {"cumsum", "cumprod", "cummin", "cummax"}:
```

Comment:
These are the only 4 that are supported in the base implementation so the `NotImplementedError` isn't needed?

Reply:
Good point, removed